aries-site/src/data/structures/tokens.js

```diff
@@ -39,6 +39,23 @@ export const tokens = [
     ],
     tags: [],
   },
+  {
+    name: 'Layout and spacing',
```

Comment:
when searching I don't see the icon next to the title 

<img width="746" alt="Screenshot 2025-02-11 at 3 05 59 PM" src="https://github.com/user-attachments/assets/2eb5bb49-d10d-4754-aa4c-55315e1c388a" />

Reply:
Huh, that's odd. Let me look into it.

Reply:
Fixed.